pytest_rpc/__init__.py

```diff
@@ -50,6 +66,36 @@ def _capture_marks(items, marks):
                     item.user_properties.append((mark, arg))
 
 
+def _get_runner(session):
+    """Gets the runner used when executing tests
+    default is 'molecule'
+
+    Args:
+        session (_pytest.main.Session): The pytest session object
+
+    Returns:
+        str: The value of the config with the highest precedence
+    """
+    #  Try to get configs from CLI and ini
+    try:
+        cli_option = session.config.getoption('--test-runner')
+    except ValueError:
+        cli_option = None
+    try:
+        ini_option = session.config.getini('test-runner')
+    except ValueError:
+        ini_option = None
+
+    # Determine if the option passed with the highest precedence is a valid option
+    highest_precedence = cli_option or ini_option or 'molecule'
+    white_list = ['molecule', 'pytest']
```

Comment:
I don't see the functional distinction here. Since molecule is a coordination abstraction that can use pytest as a verification engine, wouldn't the value for `test-runner` be `pytest` even if it was invoked by the molecule framework?